modules/tensor_mechanics/src/materials/ComputeMeanThermalExpansionEigenstrainBase.C

```diff
@@ -26,17 +26,6 @@ ComputeMeanThermalExpansionEigenstrainBase::ComputeMeanThermalExpansionEigenstra
 }
 
 void
-ComputeMeanThermalExpansionEigenstrainBase::initialSetup()
-{
-  _alphabar_stress_free_temperature = meanThermalExpansionCoefficient(_stress_free_temperature);
-  _thexp_stress_free_temperature =
-      _alphabar_stress_free_temperature * (_stress_free_temperature - referenceTemperature());
```

Comment:
Waaaait a minute, now you are not initializing those two members, but you still use them below. The solution here is to remove these member variables and compute these values on the fly below.